apps/docs/content/guides/realtime/authorization.mdx

```diff
@@ -0,0 +1,293 @@
+      ru.user_id
+    from
+      rooms_users ru
+    where
+      ru.user_id = (select auth.uid())
+      and ru.topic = (select realtime.topic())
+      and realtime.messages.extension in ('presence')
+  )
+);
+```
+
+#### Allow users to send Presence messages on a channel
+
+To update the Presence status for a user create a policy for `insert` on `realtime.messages` where the value of `realtime.messages.extension` is `presence`.
+
+```sql
+create policy "authenticated users can register and update their presence in a room"
+on "realtime"."messages"
+for insert
+to authenticated
+with check (
+  exists (
+    select
+      ru.user_id
+    from
+      rooms_users ru
+    where
+      ru.user_id = (select auth.uid())
+      and ru.name = (select realtime.topic())
+      and realtime.messages.extension in ('presence')
+  )
+);
+```
+
+### Presence and Broadcast
+
+Authorize both Presence and Broadcast by including both extensions in the `where` filter.
+
+#### Broadcast and Presence read
+
+Authorize Presence and Broadcast read in one RLS policy.
+
+```sql
+create policy "authenticated user can listen to Broadcast and Presence messages in room"
+on "realtime"."messages"
+for select
+to authenticated
+using (
+  exists (
+    select
+      ru.user_id
+    from
+      rooms_users ru
+    where
+      ru.user_id = (select auth.uid())
+      and ru.topic = (select realtime.topic())
+      and realtime.messages.extension in ('broadcast', 'presence')
+  )
+);
+```
+
+#### Broadcast and Presence write
+
+Authorize Presence and Broadcast write in one RLS policy.
+
+```sql
+create policy "authenticated user can send Broadcast and Presence messages in a room"
+on "realtime"."messages"
+for insert
+to authenticated
+with check (
+  exists (
+    select
+      ru.user_id
+    from
+      rooms_users ru
+    where
+      ru.user_id = (select auth.uid())
+      and ru.name = (select realtime.topic())
+      and realtime.messages.extension in ('broadcast', 'presence')
+  )
+);
+```
+
+## Interaction with Postgres Changes
+
+Realtime Postgres Changes are separate from Channel authorization. The `private` Channel option does not apply to Postgres Changes.
+
+When using Postgres Changes with RLS, database records are sent only to clients who are allowed to read them based on your RLS policies.
+
+## Updating RLS policies
+
+When you change an RLS policy these changes are picked up when a user connects to a Channel.
+
+When a user connects, Realtime queries your database and generates a representation of the permissions for a user for a Channel.
+
+Permissions are cached for the duration of the connection so we don't have to query your database for each Channel message.
+
+RLS policies are re-checked when access tokens are updated on the Channel. Our client libraries default to updating the access token every 30 seconds.
+
+If a user Channel permissions have been updated these changes will be picked up when the access token is sent over the Channel from the client.
```

Comment:
```suggestion
If a user Channel permissions have been updated these changes will be picked up when the refresh token is sent over the Channel from the client.
```

Reply:
Technically it's another "access token" which sent over to replace the current one used. A "refresh token" is not a thing. It's just a new "access token".